tests/PSRule.Rules.Azure.Tests/Azure.Redis.Tests.ps1

```diff
@@ -379,6 +379,26 @@ Describe 'Azure.Redis' -Tag 'Redis' {
             $ruleResult.Length | Should -Be 7;
             $ruleResult.TargetName | Should -BeIn 'redis-K', 'redis-L', 'redis-M', 'redis-N', 'redis-O', 'redis-P', 'redis-S';
         }
+
+        It 'Azure.Redis.Version' {
+            $filteredResult = $result | Where-Object { $_.RuleName -eq 'Azure.Redis.Version' };
+
+            # Fail
+            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Fail' });
+            $ruleResult | Should -Not -BeNullOrEmpty;
+            $ruleResult.Length | Should -Be 9;
+            $ruleResult.TargetName | Should -BeIn 'redis-B', 'redis-C', 'redis-D', 'redis-E', 'redis-F', 'redis-G', 'redis-H', 'redis-I', 'redis-J';
+
+            $ruleResult[0].Reason | Should -BeExactly "The Azure Cache for Redis should use the latest supported version (6.0) of Redis.";
+            $ruleResult[1].Reason | Should -BeExactly "The Azure Cache for Redis should use the latest supported version (6.0) of Redis.";
+            $ruleResult[2].Reason | Should -BeExactly "The Azure Cache for Redis should use the latest supported version (6.0) of Redis.";
+
+            # Pass
+            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Pass' });
+            $ruleResult | Should -Not -BeNullOrEmpty;
+            $ruleResult.Length | Should -Be 3;
+            $ruleResult.TargetName | Should -BeIn 'redis-A', 'redis-Q', 'redis-R';
```

Comment:
I think we should add a test case for version `6` and when `properties.redisVersion` is not specified.